cmd/console-server/main.go

```diff
@@ -0,0 +1,384 @@
+/*
+ * Copyright 2019, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"github.com/99designs/gqlgen/cmd"
+	"github.com/99designs/gqlgen/graphql"
+	"github.com/99designs/gqlgen/handler"
+	"github.com/enmasseproject/enmasse/pkg/apis/enmasse/v1beta1"
+	adminv1beta2 "github.com/enmasseproject/enmasse/pkg/client/clientset/versioned/typed/admin/v1beta2"
+	enmassev1beta1 "github.com/enmasseproject/enmasse/pkg/client/clientset/versioned/typed/enmasse/v1beta1"
+	"github.com/enmasseproject/enmasse/pkg/consolegraphql"
+	"github.com/enmasseproject/enmasse/pkg/consolegraphql/agent"
+	"github.com/enmasseproject/enmasse/pkg/consolegraphql/cache"
+	"github.com/enmasseproject/enmasse/pkg/consolegraphql/resolvers"
+	"github.com/enmasseproject/enmasse/pkg/consolegraphql/server"
+	"github.com/enmasseproject/enmasse/pkg/consolegraphql/watchers"
+	"github.com/enmasseproject/enmasse/pkg/util"
+	user "github.com/openshift/client-go/user/clientset/versioned/typed/user/v1"
+	"github.com/vektah/gqlparser/gqlerror"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/tools/clientcmd/api"
+	"log"
+	"net/http"
+	"net/http/httputil"
+	"os"
+	"reflect"
+	"time"
+)
+
+/*
+
+For development purposes, you can run the console backend outside Kubernetes:
+
+export DISABLE_PROPAGATE_BEARER=1
+export ENMASSE_ADDRESS_SPACE_TYPE_FILE=.../console//console-server/src/main/resources/addressSpaceTypes.json
+ENMASSE_ADDRESS_TYPE_FILE=.../console//console-server/src/main/resources/addressTypes.json
```

Comment:
Consider adding this to a 'run' target in the makefile in console/console-server

Reply:
I've got a few ideas to improve how to run the console-server outside the kubernetes environment for development purposes, I'll try to incorporate this idea too.

Reply:
@lulf With 307a953, for development purposes, you'll run an oc route to expose the agent(s) outside the container so that the console-server can contact them.  This behaviour is all controlled by passing the `--developmentMode=true` command line argument.  I've also added the `run` target to the Makefile, as you suggest.  Comments welcome.